sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -70,6 +77,28 @@ static void info(const char *fmt,...)
 static void info(const char *fmt,...) {}
 #endif
 
+// New function to ensure the same behaviour for random number generation on windows and linux
+// note that we always use the same seed to get reproducible results
+#if INT_MAX == 0x7FFFFFFF
+std::mt19937 mt_rand(std::mt19937::default_seed);
+#elif INT_MAX == 0x7FFFFFFFFFFFFFFF
+std::mt19937_64 mt_rand(std::mt19937::default_seed);
+#else
+info("Random number generator is not fixed for this system. Please report issue. INT_MAX=%d\n", INT_MAX);
+exit(1);
+#endif
+
+// Function to set a new seed - exposed in interface
+void set_seed(unsigned custom_seed) {
+    mt_rand.seed (custom_seed);
```

Comment:
is space between method and args conventional?

Reply:
it was like that in the C++ example that I followed so I thought so. but honestly I have no idea :)

Reply:
I'll remove the space since it seems to be more conventional to use no space in c++